### ISSUE TITLE
Client issue on connecting to any stream

### ISSUE THREAD

**Issue by scj643** [PC-Logix/OpenFM#17]
```
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: javax.sound.sampled.UnsupportedAudioFileException: file is not a supported file type
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at javax.sound.sampled.AudioSystem.getAudioFileFormat(AudioSystem.java:1045)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at pcl.OpenFM.TileEntity.TileEntityRadio.startStream(TileEntityRadio.java:99)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at pcl.OpenFM.network.RadioBlockMessageHandlerClient.onMessage(RadioBlockMessageHandlerClient.java:65)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at pcl.OpenFM.network.RadioBlockMessageHandlerClient.onMessage(RadioBlockMessageHandlerClient.java:13)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:37)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:17)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:98)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at io.netty.channel.embedded.EmbeddedChannel.writeInbound(EmbeddedChannel.java:169)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at cpw.mods.fml.common.network.internal.FMLProxyPacket.func_148833_a(FMLProxyPacket.java:77)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.client.multiplayer.PlayerControllerMP.func_78765_e(PlayerControllerMP.java:273)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1602)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.client.main.Main.main(SourceFile:148)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.reflect.Method.invoke(Method.java:497)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at java.lang.reflect.Method.invoke(Method.java:497)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:394)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:170)
[23:36:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable$WrappedPrintStream:println:748]: 	at org.multimc.EntryPoint.main(EntryPoint.java:54)
[23:36:34] [Client thread/ERROR] [FML]: SimpleChannelHandlerWrapper exception
java.lang.NullPointerException
	at pcl.OpenFM.TileEntity.TileEntityRadio.startStream(TileEntityRadio.java:107) ~[TileEntityRadio.class:?]
	at pcl.OpenFM.network.RadioBlockMessageHandlerClient.onMessage(RadioBlockMessageHandlerClient.java:65) ~[RadioBlockMessageHandlerClient.class:?]
	at pcl.OpenFM.network.RadioBlockMessageHandlerClient.onMessage(RadioBlockMessageHandlerClient.java:13) ~[RadioBlockMessageHandlerClient.class:?]
	at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:37) ~[SimpleChannelHandlerWrapper.class:?]
	at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:17) ~[SimpleChannelHandlerWrapper.class:?]
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:98) ~[SimpleChannelInboundHandler.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [DefaultChannelHandlerContext.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [DefaultChannelHandlerContext.class:?]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) [MessageToMessageDecoder.class:?]
	at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111) [MessageToMessageCodec.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [DefaultChannelHandlerContext.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [DefaultChannelHandlerContext.class:?]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785) [DefaultChannelPipeline.class:?]
	at io.netty.channel.embedded.EmbeddedChannel.writeInbound(EmbeddedChannel.java:169) [EmbeddedChannel.class:?]
	at cpw.mods.fml.common.network.internal.FMLProxyPacket.func_148833_a(FMLProxyPacket.java:77) [FMLProxyPacket.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) [ej.class:?]
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_78765_e(PlayerControllerMP.java:273) [bje.class:?]
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1602) [bao.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973) [bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898) [bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_60]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_60]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_60]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_60]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_60]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_60]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_60]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_60]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:394) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:170) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:54) [NewLaunch.jar:?]
[23:36:34] [Client thread/ERROR] [FML]: There was a critical exception handling a packet on channel openfm
java.lang.NullPointerException
	at pcl.OpenFM.TileEntity.TileEntityRadio.startStream(TileEntityRadio.java:107) ~[TileEntityRadio.class:?]
	at pcl.OpenFM.network.RadioBlockMessageHandlerClient.onMessage(RadioBlockMessageHandlerClient.java:65) ~[RadioBlockMessageHandlerClient.class:?]
	at pcl.OpenFM.network.RadioBlockMessageHandlerClient.onMessage(RadioBlockMessageHandlerClient.java:13) ~[RadioBlockMessageHandlerClient.class:?]
	at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:37) ~[SimpleChannelHandlerWrapper.class:?]
	at cpw.mods.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:17) ~[SimpleChannelHandlerWrapper.class:?]
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:98) ~[SimpleChannelInboundHandler.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) ~[DefaultChannelHandlerContext.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) ~[DefaultChannelHandlerContext.class:?]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103) ~[MessageToMessageDecoder.class:?]
	at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111) ~[MessageToMessageCodec.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) ~[DefaultChannelHandlerContext.class:?]
	at io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) ~[DefaultChannelHandlerContext.class:?]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785) ~[DefaultChannelPipeline.class:?]
	at io.netty.channel.embedded.EmbeddedChannel.writeInbound(EmbeddedChannel.java:169) ~[EmbeddedChannel.class:?]
	at cpw.mods.fml.common.network.internal.FMLProxyPacket.func_148833_a(FMLProxyPacket.java:77) [FMLProxyPacket.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) [ej.class:?]
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_78765_e(PlayerControllerMP.java:273) [bje.class:?]
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1602) [bao.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973) [bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898) [bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_60]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_60]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_60]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_60]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_60]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_60]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_60]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_60]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:394) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:170) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:54) [NewLaunch.jar:?]
[23:36:34] [Client thread/INFO] [openfm]: Stopping all currently running radio streams.
````

the file http://scj643.theender.net/music/love-the-world-string.mp3

**Comment by CaitlynMainer** [comment]
I've been looking into a fix for this for a bit... it seems forge isn't loading the SPI into the classpath properly.  current fix is to drop http://caitlynmainer.com/OpenFM/mp3spi-1.9.5-1.jar and http://caitlynmainer.com/OpenFM/vorbisspi-1.0.3-1.jar  into your mods directory client side

**Comment by CaitlynMainer** [comment]
Closed in commit 0582cb29bc1c12e58ce56437b498a2d4ff37e0ce
